<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IHMPendu</class>
 <widget class="QMainWindow" name="IHMPendu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1170</width>
    <height>654</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>IHMPendu</string>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="PenduBattons">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>301</width>
      <height>321</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="NBrErreurs">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>360</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="JeuduPenduTitre">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>20</y>
      <width>241</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>25</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Jeu Du Pendu</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="NomDuJoueur">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>80</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Nom du joueur :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="LineEditNomDuJoueur">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>110</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="Bonton_Jouer">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>110</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>JOUER</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Bonton_Config">
    <property name="geometry">
     <rect>
      <x>638</x>
      <y>160</y>
      <width>111</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Config</string>
    </property>
   </widget>
   <widget class="QLabel" name="MotaTrouver">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>220</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Mot Ã  Trouver :</string>
    </property>
   </widget>
   <widget class="QLabel" name="LabelMotaTrouver">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>270</y>
      <width>521</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="lineWidth">
     <number>2</number>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="NomduFichier">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>570</y>
      <width>311</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="MotDisponibles">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>570</y>
      <width>231</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>400</y>
      <width>971</width>
      <height>131</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="alphabet" rowminimumheight="0">
     <property name="topMargin">
      <number>1</number>
     </property>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1170</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
